client/src/common/components/ArchivedMessage.scss

```diff
@@ -7,6 +7,7 @@
   background: $bg-color;
   padding: four-by(4) four-by(8);
   box-shadow: 0 1px 3px rgba($shadow-color, 0.15);
+  max-width: 465px;
```

Comment:
How was the value chosen? Does it follow your design or it is a random value that looks ok in this case?

Reply:
A random value that looks okay in this case, we don't have any design rules to follow. 

The whole design will be re-implemented anyway in coming sprints.

Reply:
Ok